grafana-tikv-dashboard.md

```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow: The scheduler traffic on each TiKV instance in real time.
+- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

Comment:
```suggestion
- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler discard ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

Reply:
updated in https://github.com/pingcap/docs/pull/15765/files